metrics-exporter-prometheus/src/builder.rs

```diff
@@ -22,6 +22,7 @@ use hyper::{
     Response, StatusCode,
 };
 
+use hyper::http::HeaderValue;
```

Comment:
Should add this to the `cfg`-gated import below, since it's only used when push gateway support is enabled.